templates/cisco_xr_show_version.template

```diff
@@ -1,28 +1,19 @@
-Value VERSION (\S+)
+Value VERSION (\S[^\[]+)
 Value UPTIME (.+)
 Value LOCATION (\S+)
-Value HARDWARE (\S+)
+Value HARDWARE (.+)
 Value BUILD_HOST (\S+)
 
 Start
-  ^.+UTC
-  ^Cisco.+Software.+Version.+
-  ^Copyright\s\(c\).+
+  ^Cisco IOS XR Software, Version ${VERSION} 
   ^Build\s+Information: -> Build_Info
-  ^[Cc]isco\s${HARDWARE}\s.+
-  ^System\suptime\sis\s${UPTIME} -> Record
-  ^\s+$$
-  ^$$
-  ^.* -> Error "Line not found"
```

Comment:
Can you add back the error handling on here? It's a requirement to make sure we're accounting for all data.

When adding back the error handling, you'll probably need to add the other lines that were removed, and possible a few others. It doesn't have to be a match on the whole line, but the beginning of the lines would be great. I'd be more than happy to assist with any issue you have.